Tests/webcomponents/ez-subitem.html

```diff
@@ -10,6 +10,7 @@
         <script src="../../../web-component-tester/browser.js"></script>
 
         <link rel="import" href="/Resources/public/webcomponents/ez-subitem.html">
+        <link rel="import" href="/Resources/public/webcomponents/mixins/ez-sandbox-yui-component.html">
```

Comment:
no, that should no be there, the import in Resources/public/webcomponents/ez-subitem.html should be enough